src/section-adding-and-subtracting-polynomials.ptx

```diff
@@ -39,102 +45,132 @@
             </statement>
         </definition>
-        
-        <example xml:id="example-identify-polynomials">
-        <statement><p> Variables in polynomials must have non-negative integer exponents and a polynomial will never have a variable in the denominator of a fraction or under a square root (or any other radical). Identify which of the following are polynomials and which are not.
-        <ol>
-            <li>The expression <m>-2x^9-\frac{7}{13}x^3-1</m>.</li>
-            <li>The expression <m>5x^{-2}-5x^2+3</m>.</li>
-            <li>The expression <m>\sqrt{2}x-\frac{3}{5}</m>.</li>
-            <li>The expression <m>5x^3-5^{-5}x-x^4</m>.</li>
-            <li>The expression <m>\frac{25}{x^2}+23-x</m>.</li>
-            <li>The expression <m>37x^6-x+8^{\frac{4}{3}}</m>.</li>
-            <li>The expression <m>\sqrt{7x}-4x^3</m>.</li>
-            <li>The expression <m>6x^{\frac{3}{2}}+1</m>.</li>
-            <li>The expression <m>6^x-3x^6</m>.</li>
-        </ol>
-        </p></statement>
-        <solution>
-            <p>
-            <ol>
-            <li>The expression <m>-2x^9-\frac{7}{13}x^3-1</m> is a polynomial.</li>
-            <li>The expression <m>5x^{-2}-5x^2+3</m> is not a polynomial. Negative exponents on variables make the expression not a polynomial.</li>
-            <li>The expression <m>\sqrt{2}x-\frac{3}{5}</m> is a polynomial. Note that the square root is only applied to the 2.</li>
-            <li>The expression <m>5x^3-5^{-5}x-x^4</m> is a polynomial. Note that numbers <em>can</em> have negative exponents in polynomials.</li>
-            <li>The expression <m>\frac{25}{x^2}+23-x</m> is not a polynomial. Variables are not allowed to be in the denominator in polynomials.</li>
-            <li>The expression <m>37x^6-x+8^{\frac{4}{3}}</m> is a polynomial. Note that numbers <em>can</em> have fractional exponents in polynomials.</li>
-            <li>The expression <m>\sqrt{7x}-4x^3</m> is not a polynomial. Variables aren't allowed under any radical. </li>
-            <li>The expression <m>6x^{\frac{3}{2}}+1</m> is not a polynomial. Exponents must be positive integers, which <m>\frac{3}{2}</m> is not.</li>
-             <li>The expression <m>6^x-3x^6</m> is not a polynomial. Only positive integer exponents are allowed in polynomials, and <m>x</m> is not an integer since it can change.</li></ol></p></solution></example>
+        <p>For example, the coefficient of the term <m>\frac{4}{3}x^6</m> is <m>\frac{4}{3}</m>, the coefficient of the second term of the polynomial <m>x^2-5x+3</m> is <m>-5</m>, and the coefficient of the term <m>\frac{y^7}{4}</m> is <m>\frac{1}{4}</m>.</p>
+
+    <remark><p>Because variables in polynomials must have non-negative integer exponents, a polynomial will never have a variable in the denominator of a fraction or under a square root (or any other radical).</p></remark>
+
+    <exercise>
```

Comment:
Can you make the answer blanks here be a dropdown list instead of the user needing to type the words? See `section-equations-and-inequalities-as-true-false-statements.ptx` for examples. Unlike those examples (which define `$yesno` popups) you could make two, one called `$yes`, one called `$no`. The `1` or `2` at the end of these popup definitions is the index of what you intend to be the correct answer. (The `'?'` has index 0.)